src/edgeworth.jl

```diff
@@ -9,7 +9,7 @@ abstract type EdgeworthAbstract <: ContinuousUnivariateDistribution end
 skewness(d::EdgeworthAbstract) = skewness(d.dist) / sqrt(d.n)
 kurtosis(d::EdgeworthAbstract) = kurtosis(d.dist) / d.n
 
-struct EdgeworthZ{D<:UnivariateDistribution} <: EdgeworthAbstract
+@auto_hash_equals struct EdgeworthZ{D<:UnivariateDistribution} <: EdgeworthAbstract
```

Comment:
perhaps add something to the dev docs recommending that `isequal` and `hash` are defined to (and perhaps recommending AutoHasEquals.jl)? https://juliastats.org/Distributions.jl/latest/extends/#Create-New-Samplers-and-Distributions-1